app/views/katello/api/v2/content_view_environments/show.json.rabl

```diff
@@ -3,6 +3,7 @@ extends 'katello/api/v2/common/identifier'
 
 extends 'katello/api/v2/common/timestamps'
 attributes :default_environment? => :default
+attributes :id
```

Comment:
Same here.

Reply:
How do I test this one?

Reply:
hammer content-view-environment doesn't have a `show` subcommand, so testing with hammer may not be possible there? but could see if `hammer -d content-view-environment list` shows it in the JSON.

Reply:
I see the `id` in the response:

```
 [2] {
                          "default" => false,
                               "id" => 4,    <------------------------ content view environment id
                             "name" => "stage/cv1",
                            "label" => "stage/cv1",
                       "created_at" => "2025-03-26 17:26:32 UTC",
                       "updated_at" => "2025-03-26 17:26:32 UTC",
                     "organization" => {
                 "name" => "Default Organization",
                "label" => "Default_Organization",
                   "id" => 1                      <---------------------- organization id
            },
```

Reply:
Result from master branch:
```
       "results" => [
        [ 0] {
                          "default" => true,
                               "id" => 1,
                             "name" => "Library",
                            "label" => "Library",
                       "created_at" => "2025-04-01 19:57:52 UTC",
                       "updated_at" => "2025-04-01 19:57:52 UTC",
                     "organization" => {
                 "name" => "Default Organization",
                "label" => "Default_Organization",
                   "id" => 1
            },
 ```